### PR TITLE
Login fe

### PR DESCRIPTION
**Pull request by Catarina98**

### CODE REVIEW

File: GrocifyApp.API/Controllers/AuthController.cs [hunk 1]
```diff
@@ -105,6 +105,7 @@ public async Task<ActionResult<string>> Login([FromBody] LoginRequestModel user)
             string token = CreateToken(userRequestModel);
 
             return Ok(token);
+            //return token;
```
Comment (jessywork) [inline GrocifyApp.API/Controllers/AuthController.cs]: remove

File: GrocifyApp.UI/src/App.jsx [hunk 2]
```diff
@@ -1,49 +1,25 @@
...
-                    <tr key={forecast.date}>
-                        <td>{forecast.date}</td>
-                        <td>{forecast.temperatureC}</td>
-                        <td>{forecast.temperatureF}</td>
-                        <td>{forecast.summary}</td>
-                    </tr>
-                )}
-            </tbody>
-        </table>;
+    const isAuthenticated = localStorage.getItem('token'); // Check if the user is authenticated
```
Comment (jessywork) [inline GrocifyApp.UI/src/App.jsx]: put the comment above this line

File: GrocifyApp.UI/src/App.jsx [hunk 3]
```diff
@@ -1,49 +1,25 @@
...
 
     return (
-        <div>
-            <h1 id="tabelLabel">Weather forecast</h1>
-            <p>This component demonstrates fetching data from the server.</p>
-            {contents}
-        </div>
+        <Router>
+            <div className="App">
```
Comment (jessywork) [inline GrocifyApp.UI/src/App.jsx:13]: why do you need this div?

**Review by jessywork — CHANGES_REQUESTED** [review]

File: GrocifyApp.UI/src/App.jsx (hunk 3 above)
Comment (Catarina98) [reply GrocifyApp.UI/src/App.jsx:13]: later in css we will use that class

**Review by jessywork — APPROVED** [review]

**Pull request merged**